guidance/models/transformers/_transformers.py

```diff
@@ -177,8 +198,6 @@ def _tokenizer(self, model, **kwargs) -> Union[
             tokenizer is not None
         ), "You must give a model name when you provide a tokenizer object!"
 
-        return tokenizer
```

Comment:
Did you mean to remove this?

Reply:
Whoops! I'll add that back in + get rid of that print that I didn't remove